packages/blocks-ui/src/panels/blocks-listing.js

```diff
@@ -9,6 +9,70 @@ import { useEditor } from '../providers/editor'
 import { useBlocks } from '../providers/blocks'
 import { useScope } from '../providers/scope'
 
+// used as a wrapper around like block types
+const BlockSorting = ({ title, children }) => {
+  const [toggle, setToggle] = useState(true)
+  const [height, setHeight] = useState()
+  const ref = useRef(null)
+  useEffect(() => {
+    setHeight(ref.current.offsetHeight)
+  }, [ref])
+  return (
+    <div
+      ref={ref}
+      sx={{
+        boxSizing: 'border-box',
+        width: '100%',
+        border: '1px solid highlight',
+        background: '#F6F6F6', // replace this with 'muted' once it's added to the theme
+        padding: '3',
+        marginBottom: '3',
+        overflow: 'hidden',
+        height: '100%',
+        maxHeight: toggle ? `${height + 52}px` : '52px',
+        transition: 'max-height .5s'
+      }}
+    >
+      <div
+        sx={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center'
+        }}
+      >
+        <h5 sx={{ margin: 0, marginBottom: 3, color: 'primary' }}>{title}</h5>
```

Comment:
I'd get rid of the double marginBottom on this element and div element just next to it, and just put a single `marginTop` on the accordion body content.

Reply:
Also, the font size of the group count is smaller than the group title. I feel like they should be the same, or the count should be smaller.